app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception, except: :not_found
+  rescue_from StandardError, with: :internal_server_error
```

Comment:
Does this still capture to Raven?

Reply:
It does not, I have addressed it in the method below, great spot!